package Throw;

public class Throwex {

	public static void main(String[] args) {
		int i=20;
		int j=0;
		try 
		{
			j=18/i;
			if(j==0) 
			{
				throw new ArithmeticException("The value of j is zero");
			}
		}
		catch(ArithmeticException e)
		{
			i=18/1;
			System.out.println("The Exception occurred,"+e);
			
		}

	}

}
